app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    @comment = post.comments.new(author: current_user, **comment_param)
+    if @comment.save
+      redirect_to user_post_path(current_user, post)
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def comment_param
+    params.require(:comment).permit(:text)
+  end
```

Comment:
- [ ] Please the comments controller is to have new and create actions as you did in the post controller.